packages/frontend/src/components/common/ResourceView/resourceUtils.ts

```diff
@@ -50,7 +50,9 @@ export const getResourceUrl = (projectUuid: string, item: ResourceViewItem) => {
         case ResourceViewItemType.CHART:
             return `/projects/${projectUuid}/saved/${item.data.uuid}`;
         case ResourceViewItemType.SPACE:
-            return `/projects/${projectUuid}/spaces/${item.data.uuid}`;
+            return `/projects/${projectUuid || item.data.projectUuid}/spaces/${
+                item.data.uuid
+            }`;
```

Comment:
in `ResourceViewGrid` we set some props to empty strings by default:
```
  pinnedItemsProps = { projectUuid: '', pinnedListUuid: '' },
```

There could be another way of solving this bug but this is a quick fix for now. Essentially we should accommodate no empty strings for space items? What do we think?

Reply:
@yasmin-e was this a breaking change from the backend changes? maybe it should be fixed in the backend?

Reply:
Looks like a FE issue where the projectUuid property is not being passed to this component. Usually, the project uuid is part of the url.

Reply:
Absolutely @ZeRego, as you've mentioned it was a props issue. 

Have done this instead, what do we think? https://github.com/lightdash/lightdash/pull/5410/commits/dd26f9fe00a07d41b24100d5b87b52bfc04fbb2a

(cc @IrakliJani , @yasmin-e )